doc/src/learn/architecture/data_mgmt.md

```diff
@@ -0,0 +1,113 @@
+
+A Full Node [can be configured](../../build/snapshot.md) to generate a state snapshot at the end of each epoch. MystenLabs is managing the RocksDB snapshots in AWS S3, and we’re encouraging the rest of the community to bring up other snapshots to ensure stronger data availability.
+
+## Full Node pruning policies
+As described above, sustainable disk usage requires FNs to prune the information about 1) historic object versions, as well as 2) transactions with the corresponding effects and events, including old checkpoint data.
+
+Both transaction and object pruners are running in the background. The logical deletion of entries in RocksDB is ultimately triggering the physical compaction of data on disk, which is governed by RocksDB background jobs: the pruning effect on disk usage is not immediate and might take multiple days.
+
+Object pruning policies are described [here](../../build/fullnode.md#object-pruning). The pruner can be either applied in the:
+* “aggressive” manner (`num-epochs-to-retain: 0`): old object versions are pruned as soon as possible, or in the
+* “epoch-based” manner (`num-epochs-to-retain: X`): old object versions are pruned after X epochs.
+_In our experience, an aggressive pruner is operating more efficiently, and is a preferable option._
+
+Transaction pruning policies are described [here](../../build/fullnode.md#transaction-pruning). Transaction pruning is defined via the `num-epochs-to-retain-for-checkpoints: X` config option: the checkpoints, including their transactions, effects and events are pruned up to X epochs ago. The preferable option for the most efficient pruner operations is 2 epochs for transaction pruning, unless the node requires keeping more historic information in local disk for RPC purposes.
+
+The FNs, which are uploading information to the archives, should be configured to ensure that pruning doesn’t occur until the corresponding information has been archived (`use-for-pruning-watermark: true`).
+
+## Full Node key-value store backup
+In order to enable historic data queries for the FNs that prune old transactional data, FN RPC logic is configured to fallback for querying transactional data from a remote store.
+In case the information about the transaction digest, effects, events, or checkpoints is not available locally, a FN retrieves the historical data from a cloud-based key-value store. This fallback retrieval is transparent to the FN clients. Note that the current implementation doesn’t keep historic object versions, and is applied to historic transaction and checkpoint information only. *_We're currently working on a similar setup for retrieving the historic object versions._
+
+
+## Pruning policy examples
```

Comment:
@phoenix-o @sadhansood could you please pay special attention to these examples? (just want to verify that they're covering our current guidance and are technically correct).

Reply:
The overall examples look correct, but the current pruning configuration requires more values to be specified. A complete example:
```
authority-store-pruning-config:
  num-latest-epoch-dbs-to-retain: 3
  epoch-db-pruning-period-secs: 3600
  max-checkpoints-in-batch: 10
  max-transactions-in-batch: 1000

  # Prune historic object versions
  num-epochs-to-retain: 0
  # Prune historic transactions of the past epochs
  num-epochs-to-retain-for-checkpoints: 2
  periodic-compaction-threshold-days: 1
```
However, we can make the repeating parameters optional by providing reasonable defaults

Reply:
The example makes sense to me.

Reply:
Thanks! Does it look good to you to land now?